airbyte-config/models/src/main/resources/types/FailureReason.yaml

```diff
@@ -0,0 +1,49 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/FailureReason.yaml
+title: FailureSummary
+type: object
+required:
+  - failureSource
+  - failureType
+  - timestamp
+additionalProperties: false
+properties:
+  failureSource:
+    description: Indicates where the error originated. If not set, the source of error is not well known.
+    type: string
+    enum:
+      - unknown
+      - source
+      - destination
+      - replicationWorker
+      - persistence
+      - normalization
+      - dbt
+  failureType:
+    description: Categorizes well known errors into types for programmatic handling. If not set, the type of error is not well known.
+    type: string
+    enum:
+      - unknown
+      - permissionError
+      - systemRestart
+      - zombie
+      - manualCancellation
```

Comment:
this isn't actually an error, right? Is it just included here because it's surfaced via an exception?

Reply:
Yeah there was a bit of back and forth on this in the tech spec. It might make sense to leave it out of the failure summary and instead just have a top-level `cancelledBy` column on the attempts table for audit-trail purposes. 

@cgardens now that you see it in code, how do you feel about it? I personally think it feels a bit unintuitive as-is. I think this is what you were getting at in the tech spec thread, like in order for this approach to make sense, we would want to use a `failed` status for cancelled jobs rather than having an entirely-separate `cancelled` status.

Reply:
Parker, I agree with what you suggested. Let's do it.